### PR TITLE
#744 administrative panel granting administrator rights

### PR DESCRIPTION
**Pull request by OlesiaBarannik**

### CODE REVIEW

File: BackEnd/administration/views.py [hunk 1]
```diff
@@ -122,3 +123,18 @@ class AutoModerationHoursView(RetrieveUpdateAPIView):
...
+
+class ModerationEmailView(RetrieveUpdateAPIView):
+    """
+    View for retrieving and updating the ModerationEmail instance.
+    Requires the user to be a superuser.
+    """
+
+    permission_classes = [IsSuperUser]
+
+    queryset = ModerationEmail.objects.all()
```
Comment (YanZhylavy) [inline BackEnd/administration/views.py:136]: This line with queryset is redundant. If you override get_object() the way you did, setting queryset simply does nothing.

Comment (OlesiaBarannik) [reply BackEnd/administration/views.py]: fixed it, thanks

File: FrontEnd/src/components/adminPage/detail-view/ModerationEmail.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import css from './ModerationEmail.module.css';
+
+const ModerationEmail = () => {
+    const fetcher = url => axios.get(url).then(res => res.data);
+    const url = `${process.env.REACT_APP_BASE_API_URL}/api/admin/email/`;
+    const { data, mutate } = useSWR(url, fetcher);
+
+    // Define email and setEmail using useState
+    const [email, setEmail] = useState(null);
+    const [error, setError] = useState(null);
```
Comment (YanZhylavy) [inline FrontEnd/src/components/adminPage/detail-view/ModerationEmail.jsx:15]: Is the error state necessary in this component? I don't see it being set to anything other than null in the code below.

Comment (OlesiaBarannik) [reply FrontEnd/src/components/adminPage/detail-view/ModerationEmail.jsx]: Fixed, thank you

**Review by Lvyshnevska — APPROVED** [review]

**Review by YanZhylavy — APPROVED** [review]

**Pull request merged**